test/e2e/pull_test.go

```diff
@@ -84,6 +84,36 @@ var _ = Describe("Podman pull", func() {
 		Expect(session).Should(Exit(0))
 	})
 
+	It("podman pull and run on split imagestore", func() {
+		tmpDir, err := os.MkdirTemp("", "")
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
Do not create your own tempdir, this is not removed so you leak things forever, use `podmanTest.TempDir`.